test/extended/image_ecosystem/mysql_replica.go

```diff
@@ -90,7 +95,7 @@ func replicationTestFactory(oc *exutil.CLI, tc testCase) func() {
 		err = oc.Run("new-app").Args("-f", tc.TemplatePath).Execute()
 		o.Expect(err).NotTo(o.HaveOccurred())
 
-		err = oc.Run("new-app").Args("-f", helperTemplate, "-p", fmt.Sprintf("DATABASE_SERVICE_NAME=%s", helperName)).Execute()
+		err = oc.Run("new-app").Args("-f", helperTemplate, "-p", fmt.Sprintf("MYSQL_VERSION=%s", tc.Version), "-p", fmt.Sprintf("DATABASE_SERVICE_NAME=%s", helperName)).Execute()
```

Comment:
The helper template is mysql-ephemeral, which is a bit confusing, [here](https://github.com/openshift/origin/blob/master/test/extended/image_ecosystem/mysql_replica.go#L39)